src/locales/data.json

```diff
@@ -0,0 +1,262 @@
+    "seeLess": "See less",
+    "seeMore": "See more",
+    "roleCreatedSuccessfully": "You have successfully created a new role",
+    "createAnotherRole": "Create another role",
+    "granularPermissionsWillBeCopied": "Only granular permissions will be copied into a custom role (for example, approval:requests:read). Wildcard permissions will not be copied into a custom role (for example, approval:*:read).",
+    "selectState": "Select a state",
+    "selectResources": "Select resources",
+    "reviewRoleDetails": "Review and confirm the details for your role, or click Back to revise.",
+    "createRole": "Create role",
+    "defineCostResources": "Define Cost Management resources",
+    "applyCostPermissionText": "Specify where you would like to apply each cost permission selected in the previous step, using the dropdown below.",
+    "createRoleFromScratch": "Create a role from scratch",
+    "copyAnExistingRole": "Copy an existing role",
+    "permissionsAddedSuccessfully": "You have successfully added permissions to the role",
+    "addedPermissions": "Added permissions",
+    "assignAtLeastOneResource": "You need to assign at least one resource to each permission.",
+    "resourcesAvailable": "Resources available for the permission",
+    "resourcesDefined": "Resources defined for the permission",
+    "filterByResource": "Filter by resource...",
+    "exitEditResourceDefinitions": "Exit edit resource definitions?",
+    "changesWillBeLost": "All changes will be lost",
+    "editResourceDefinitions": "Edit resource definitions",
+    "editPermissionsUsingArrows": "Give or remove permissions to specific resources using the arrows below.",
+    "roleWithNameExists": "Role with this name already exists.",
+    "deleteRoleQuestion": "Delete role?",
+    "defineAtLeastOneResource": "At least one resource must be defined for this permission",
+    "permissionWillNotBeGrantedThroughRole": "The <b>{permission}</b> permission will no longer be granted through the <b>{role}</b> role.",
+    "permissionsWillNotBeGrantedThroughRole": "The <b>{permissions}</b> permissions will no longer be granted through the <b>{role}</b> role.",
+    "lastCommit": "Last commit",
+    "resourceDefinitionsApplyToCost": "Resource definitions only apply to Cost Management permissions",
+    "removePermissionQuestion": "Remove permission?",
+    "removePermission": "Remove permission",
+    "removePermissionsQuestion": "Remoe permissions?",
+    "removePermissions": "Remove permissions",
+    "noRolePermissions": "There are no permissions in this role",
+    "definedResources": "Defined resources",
+    "userDescription": "{username}'s roles, groups and permissions.",
+    "userNotFound": "User not found",
+    "userNotFoundDescription": "User with username {username} does not exist.",
+    "addNewUsersText": "To add new users or manage existing users, go to your"
+  }
+}
```

Comment:
I don't think this file is needed

Reply:
I think it is needed before we get the memsource to create it for us. I think in the first step we will only get one file per language

Reply:
In that case shouldn't this be auto-generated too? And I think, if not this only works if english is the only language

Reply:
This already seems to be out of sync with the auto-generated data.json there are 266 strings there and 259 here